velox/docs/functions/spark/comparison.rst

```diff
@@ -0,0 +1,70 @@
+=====================================
+Comparison Functions
+=====================================
+
+.. js:function:: between(x, min, max) -> boolean
+    :noindex:
+
+    Returns true if x is within the specified [min, max] range
+    inclusive. The types of all arguments must be the same.
+    Supported types are: TINYINT, SMALLINT, INTEGER, BIGINT, DOUBLE, REAL.
+
+.. function:: equalnullsafe(x, y) -> boolean
+
+    Returns true if x is equal to y. Supports all scalar types. The
+    types of x and y must be the same. this differs from EqualTo in that
+    it returns true (rather than NULL) if both inputs are NULL,
+    and false (rather than NULL) if one of the input is NULL and the other is not NULL.
+    Function equalnullsafe corresponds to scala's operator ``<=>``.
+
+.. function:: equalto(x, y) -> boolean
+
+    Returns true if x is equal to y. Supports all scalar types. The
+    types of x and y must be the same. Function equalto corresponds
+    to scala's operator ``=`` and ``==``.
+
+.. function:: greaterthan(x, y) -> boolean
+
+    Returns true if x is greater than y. Supports all scalar types. The
+    types of x and y must be the same. Function greaterthan corresponds to scala's operator ``>``.
+
+.. function:: greaterthanorequal(x, y) -> boolean
+
+    Returns true if x is greater than y or x is equal to y. Supports all scalar types. The
+    types of x and y must be the same. Function greaterthanorequal corresponds to scala's operator ``>=``.
+
+.. js:function:: greatest(value1, value2, ..., valueN) -> [same as input]
+    :noindex:
+
+    Returns the largest of the provided values. skipping null values. Supports all scalar types. 
+    The types of all arguments must be the same. ::
+
+        SELECT greatest(10, 9, 2, 4, 3); -- 10
+        SELECT greatest(10, 9, 2, 4, 3, null); -- 10
+
+.. js:function:: least(value1, value2, ..., valueN) -> [same as input]
+    :noindex:
+
+    Returns the smallest of the provided values. Skipping null values. Supports all scalar types.
```

Comment:
```
Returns the smallest of the input values ignoring NULLs.
```

Reply:
It's really an interesting situation.
I will add this example.
select least(null ,null) - null